src/main/webapp/app/entities/drag-and-drop-mapping/drag-and-drop-mapping-detail.html

```diff
@@ -13,6 +13,10 @@ <h2><span data-translate="artemisApp.dragAndDropMapping.detail.title">Drag And D
             <span>{{vm.dragAndDropMapping.dropLocationIndex}}</span>
         </dd>
         <dt><span data-translate="artemisApp.dragAndDropMapping.dragItem">Drag Item</span></dt>
+        <dt><span data-translate="artemiSApp.dragAndDropMapping.invalid">Invalid</span></dt>
```

Comment:
I think you made a typo when doing the Find & Replace for artemisApp, because there are a lot of instances where the "s" is uppercase, although it should be lowercase. Please run Find & Replace again and replace all instances of "artemiSApp" with "artemisApp".